libraries/AC_Fence/AC_PolyFence_loader.h

```diff
@@ -7,13 +7,21 @@
 
 #define AC_POLYFENCE_FENCE_POINT_PROTOCOL_SUPPORT 1
 
+// CIRCLE_INCLUSION_INT stores the radius an a 32-bit integer in
+// metres.  This was a bug, and CIRCLE_INCLUSION was created to store
+// as a 32-it float instead.  We save as _INT in the case that the
```

Comment:
small typo in the comments, "32-**b**it"

Reply:
Fixed.